shared/scss/components/ckan/dataset-list.scss

```diff
@@ -85,6 +85,10 @@
       margin: 0;
     }
   }
+
+  .dataset-item-link {
+    display: block;
+  }
```

Comment:
Sorry, you already approved the earlier changes but had to change this from inline to block for the cursor to behave more coherently.